.github/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+## Description
+
+<!-- What does this PR do? Please list any issue that these changes address and how you have gone about implementing the changes -->
+
+## Checklist
+
+I confirm that I have completed the following checks:
+
+**Testing**
+- [ ] If I have changed any of the Python or Fortran code, I have rebuilt and run the tests locally and they have passed.
```

Comment:
Is this necessary? The CI will do this.

Reply:
I saw this more as a reminder to check your code *before* pushing it and having it rejected by the CI. I will make this a comment rather than a tickbox I think.